plane/source/docs/roll-pitch-controller-tuning.rst

```diff
@@ -18,11 +18,10 @@ new aircraft, as everything else relies on getting this right.
 
    The default values for the
    roll and pitch controllers in Plane are quite deliberately too small for
-   most aircraft. This is because small values will cause APM to not
+   most aircraft. This is because small values will cause autopilot to not
```

Comment:
ArduPilot instead of autopilot, or "the autopilot"

Reply:
done